recipes/macfsevents/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "macfsevents" %}
```

Comment:
If you change this to `MacFSEvents`, which is what the pypi name is, I think the download will start working.